lib/rules/block-scoped-var.js

```diff
@@ -316,3 +316,5 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.schema = [];
```

Comment:
Wouldn't it be easier to just check if there's a schema, then validate, otherwise skip?